intelmq/bots/parsers/shadowserver/parser.py

```diff
@@ -86,8 +86,8 @@ def parse_line(self, row, report):
         conf = self._sparser_config
 
         # https://github.com/certtools/intelmq/issues/1271
-        if conf == config.drone and row.get('infection') == 'spam':
-            conf = config.drone_spam
+        # if conf == config.drone and row.get('infection') == 'spam':
+        #    conf = config.drone_spam
```

Comment:
Was the feed changed? If yes, please remove the lines entirely.

Reply:
Updated in a2aeda778d7e139eae453d608114b4f1fb017636